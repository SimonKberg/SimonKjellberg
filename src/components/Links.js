// @flow strict
import * as React from 'react'

import Heading from './Heading'
import Link from './Link'
import * as UnorderedList from './UnorderedList'

const Chat = () => (
  <section>
    <Heading level="2">Links</Heading>
    <UnorderedList.List>
      <UnorderedList.ListItem>
        <Link href="https://github.com/simonkberg">GitHub</Link>
      </UnorderedList.ListItem>
      <UnorderedList.ListItem>
        <Link href="https://linkedin.com/in/simonkjellberg">LinkedIn</Link>
      </UnorderedList.ListItem>
    </UnorderedList.List>
  </section>
)

export default Chat
