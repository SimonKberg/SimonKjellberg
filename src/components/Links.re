[@bs.config {jsx: 3}];

open Utils;

[@react.component]
let make = () => {
  <section>
    <Heading level=`Level2> {"Links" |> str} </Heading>
    <UnorderedList>
      <UnorderedListItem>
        <Link href="https://github.com/simonkberg"> {"GitHub" |> str} </Link>
      </UnorderedListItem>
      <UnorderedListItem>
        <Link href="https://linkedin.com/in/simonkjellberg">
          {"LinkedIn" |> str}
        </Link>
      </UnorderedListItem>
    </UnorderedList>
  </section>;
};

let default = make;
