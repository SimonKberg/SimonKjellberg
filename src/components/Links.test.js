import * as React from 'react'
import renderer from 'react-test-renderer'
import Links from './Links.bs'

test('<Links />', () => {
  const tree = renderer.create(<Links />).toJSON()
  expect(tree).toMatchSnapshot()
})
