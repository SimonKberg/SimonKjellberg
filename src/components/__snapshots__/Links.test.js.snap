// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Links /> 1`] = `
.emotion-1 {
  color: currentColor;
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-1:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-0 {
  position: relative;
  margin-top: 1em;
  margin-bottom: 1em;
  margin-left: 1.5ch;
  font-size: 1rem;
  font-weight: 700;
}

.emotion-0::before {
  position: absolute;
  left: -1.5ch;
  content: ">";
  color: #c0c0c0;
}

.emotion-5 {
  list-style-type: none;
  padding: 0;
  margin-top: 1em;
  margin-bottom: 1em;
  margin-left: 1.5ch;
}

.emotion-2 {
  position: relative;
}

.emotion-2::before {
  content: "*";
  position: absolute;
  left: -1.5ch;
  color: #c0c0c0;
}

.emotion-2 > .emotion-5 {
  margin-top: 0;
  margin-bottom: 0;
}

<section>
  <h2
    className="emotion-0"
  >
    Links
  </h2>
  <ul
    className="emotion-5"
  >
    <li
      className="emotion-2"
    >
      <a
        className="emotion-1"
        href="https://github.com/simonkberg"
      >
        GitHub
      </a>
    </li>
    <li
      className="emotion-2"
    >
      <a
        className="emotion-1"
        href="https://linkedin.com/in/simonkjellberg"
      >
        LinkedIn
      </a>
    </li>
  </ul>
</section>
`;
